scanner/Makefile

```diff
@@ -48,12 +48,14 @@ endif
 BUILD_FLAGS := CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH}
 BUILD_CMD := go build -trimpath -ldflags="-X github.com/stackrox/stackrox/scanner/internal/version.Version=$(TAG)" -o image/scanner/bin/scanner ./cmd/scanner
 
+DOCKERBUILD := $(CURDIR)/../scripts/docker-build.sh
+
 ###############################################################
 ###### Binaries we depend on (need to be defined on top) ######
 ###############################################################
 
 OSSLS_BIN := $(GOBIN)/ossls
-$(OSSLS_BIN): deps
+$(OSSLS_BIN):
```

Comment:
Should this be behind `.PHONY`? It is executed when no targets are passed:

```sh
$ make 
+ /Users/dcaravel/dev/stackrox/stackrox/scanner/.gobin/ossls
```

Reply:
I never knew PHONY helped with that!

Reply:
ah ok it's not because of PHONY. It's because `make` runs the first target, by default. I will add the conventional "all"